stage1_fly/aci/aci-manifest.in

```diff
@@ -28,6 +28,14 @@
         {
             "name": "coreos.com/rkt/stage1/gc",
             "value": "/gc"
+        },
+        {
+            "name": "coreos.com/rkt/stage1/stop",
+            "value": "/stop_fly"
+        },
+        {
+            "name": "coreos.com/rkt/stage1/interface-version",
+            "value": "3"
```

Comment:
Again fly biting us, since it's not a real stage1. I'd not include the version in `fly` stage1